src/lando/main/scm/hg.py

```diff
@@ -289,6 +289,14 @@ def apply_patch(
                 + ["--logfile", f_msg.name]
             )
 
+    def apply_patch_bytes(self, patch_bytes: bytes):
+        """Apply the given `hg export` to the repo directly."""
+        with tempfile.NamedTemporaryFile(mode="wb", suffix=".patch") as tmp_file:
+            tmp_file.write(patch_bytes)
+            tmp_file.flush()
+
+            self.run_hg(["import", tmp_file.name])
+
```

Comment:
Wondering what the value is of continuing to add implementations for `hg` (particularly in the context of automation) given that there is no plan to use these, and we also decommissioned the unused worker? I also don't think this code path is tested in this PR.

This should probably just raise `NotImplementedError`.